tapgarden/planter.go

```diff
@@ -204,6 +205,81 @@ func (p *PendingAssetGroup) PSBT(
 	return packet, nil
 }
 
+// DeserializeWitnessStack deserializes a serialized witness stack into a
+// [][]byte.
+//
+// TODO(ffranr): Where should this function live? Do we have an existing
+// function that does this?
+func DeserializeWitnessStack(serialized []byte) ([][]byte, error) {
```

Comment:
If you use `wire.TxWitness`, then you can use the encode/decode methods from there: https://pkg.go.dev/github.com/btcsuite/btcd/wire#TxWitness

Reply:
Hmm, or actually looks like not much of it is exported: https://github.com/btcsuite/btcd/blob/v0.24.2/wire/msgtx.go#L588-L626

Reply:
`DeserializeWitnessStack` removed in latest commits.